api/src/wfl/service/terra.clj

```diff
@@ -1,10 +1,14 @@
 (ns wfl.service.terra
   "Analyze data in Terra using the Firecloud/Terra API."
   (:require [clojure.data.json :as json]
+            [clojure.java.io :as io]
             [clj-http.client :as http]
             [wfl.once :as once]
             [wfl.util :as util]))
 
+(defn ^:private json-body [response]
```

Comment:
This is now defined in a few places. Just move the original (i think in pubsub.clj) to a common location (utils?)

Reply:
I see you have moved it to `util` module in a previous PR, will leverage that!